src/mpi/romio/adio/common/ad_write_str.c

```diff
@@ -133,7 +133,7 @@ void ADIOI_GEN_WriteStrided(ADIO_File fd, const void *buf, int count,
     ADIO_Offset new_bwr_size, new_fwr_size, st_fwr_size, fwr_size=0, bwr_size, req_len;
     static char myname[] = "ADIOI_GEN_WriteStrided";
 
-    if (fd->hints->ds_write == ADIOI_HINT_DISABLE) {
+    if (1 || fd->hints->ds_write == ADIOI_HINT_DISABLE) {
```

Comment:
uh... this shouldn't be here

Reply:
I think the problem is that you are reviewing those patches one at a time.
can you just look at the github diff as if it's just 1 squashed commit?
On this page you can see just the changes:
https://github.com/pmodels/mpich/pull/3651/files

there are several intermediate patches or some accidental patches committed like this one that i reverted later.. i don't think you should be reviewing those, but just the final patch since you are going to squash it anyway, no?